app/main/views/g_cloud.py

```diff
@@ -653,6 +654,40 @@ def which_service_won_contract(framework_family, project_id):
     ), 200 if not errors else 400
 
 
+@direct_award.route(
+    '/<string:framework_family>/projects/<int:project_id>/tell-us-about-contract',
+    methods=['GET', 'POST']
+)
+def tell_us_about_contract(framework_family, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_family)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    if not is_direct_award_project_accessible(project, current_user.id):
+        abort(404)
+
+    if not project['lockedAt']:
+        abort(400)
+
+    form = TellUsAboutContractForm()
+
+    if form.validate_on_submit():
+        flash(Markup(f"""You've updated '{project['name']}'"""), 'success')
```

Comment:
@lfdebrux We need to fix this before it gets out to production - wrapping this in Markup means we need to escape the project name, or this can happen:
![screen shot 2018-06-08 at 19 19 17](https://user-images.githubusercontent.com/2920760/41174050-df39337c-6b50-11e8-9c3e-7fa840f4a154.png)

Reply:
Probably all we need to do in this case is not wrap it in `Markup`.

Reply:
Just saw this.

Wow. That's a real howler. I am incredibly embarrassed.

That will teach me to copy paste without thinking :/

Reply:
No need to be embarrassed. Easy thing to miss and it's unlikely to happen again.

Been trying to think how we could have a test make sure this doesn't happen on a broader scale than writing a custom test for each page, but not managed to come up with a good idea yet. Some 'dangerous' API stubs that are easy to inject into tests might be a good start. Maybe a process in the manual, as well, listing common vulnerabilities that should be tested against when implementing a new page. Something to think about.